config/config_handler.go

```diff
@@ -198,9 +219,10 @@ func (ch *pdConfHandler) run() {
 
 			ch.reloadFunc(ch.curConf.Load().(*Config), newConf)
 			ch.curConf.Store(newConf)
-			logutil.Logger(context.Background()).Info("PDConfHandler update config successfully",
-				zap.String("fromVersion", ch.version.String()), zap.String("toVersion", version.String()))
 			ch.version = version
+			logutil.Logger(context.Background()).Info("PDConfHandler update config successfully",
+				zap.String("fromVersion", ch.version.String()), zap.String("toVersion", version.String()),
+				zap.String("new_config", newConfContent))
```

Comment:
s/ newConfContent/ newConf ?
or
We only need to log the changed conf items?

Reply:
Now we have no method to check which config items changed, so I log them all.
But I will implement it in the next few PRs.